Source/Resource/Plugin/Language/eu.csv

```diff
@@ -1162,12 +1161,12 @@ SamplesPerFrame;
 SamplingCount;Lagin zenbatekoa
 SamplingRate;Laginketa neurria
 Save;Gorde
-ScanOrder;Mihaketa hurrenkera

```

Comment:
Current translation is so wrong? (several times, when you remove completely a translation without a replacement)

Reply:
Some of the more technical terms were not well translated and, in the case of those that are not usually used by non-expert users, the original term is usually maintained.